tests/testthat/test_run_suite.R

```diff
@@ -1,19 +1,19 @@
 context("check quality suite tests")
 
 test_that("runSuite() works", {
-  metadataXML <- system.file("extdata/example_EML.xml", package = "metadig")
+  metadataFILE <- system.file("extdata/example_EML.xml", package = "metadig")
   suiteXML <- system.file("extdata/example_suite.xml", package = "metadig")
   dirXML <- system.file("extdata", package = "metadig")
   sysmetaXML <- system.file("extdata/example_sysmeta.xml", package = "metadig")
 
   expect_error(runSuite(7, 7, 7, 7))
   expect_error(runSuite(suiteXML, 7, 7, 7))
   expect_error(runSuite(suiteXML, dirXML, 7, 7))
-  expect_error(runSuite(c(suiteXML, dirXML), dirXML, metadataXML))
-  expect_error(runSuite(suiteXML, c(suiteXML, dirXML), metadataXML))
+  expect_error(runSuite(c(suiteXML, dirXML), dirXML, metadataFILE))
```

Comment:
can you add the test json to the test data directory, and update this check so that it works correctly? I think it will either require restructuring that directory, or rewriting this test, or both. when I added the json the test failed I think because it is interpreting it as a check not a document to be checked

Reply:
I added it as an example dataset in `extdata`, but the issue is not that it thinks the new json file is one of the check XML files (there are still only the 4 that end up in the `suite` value, see below. The issue is that I have only updated the title length check XML file to have a jq command for the schema.org JSON. So all of the other checks fail on the JSON right now.

```
basename(suite)
[1] "dataset_title_length-check.xml"         "datatype_check.xml"                    
[3] "entity_attributes_sufficient_check.xml" "methods_present.xml"
```

Reply:
The tests won't fail for now but once we get the `jq` commands added to the other check XML files, we will need to remove the code I added here: https://github.com/NCEAS/metadig-r/pull/9/commits/48b6e785fe9ea003d2728037e3a82bc51765a21e